spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -183,9 +183,16 @@ def submit_prefilled_otp_code
       sign_in_before_2fa(user)
 
       code = RecoveryCodeGenerator.new(user).create
+      code_words = code.split(' ')
       click_link t('devise.two_factor_authentication.recovery_code_fallback.link')
-      fill_in 'code', with: code
-      click_button t('forms.buttons.submit.default')
+
+      fields = page.all('input[type="text"]')
+
+      fields.size.times do |index|
```

Comment:
I think its repeated twice, normally I wouldn't break it out at this point but I will add it to the session_helper file!